src/murmur/Cert.cpp

```diff
@@ -116,6 +117,27 @@ void Server::initializeCert() {
 		qlCA = ql;
 	}
 
+#if defined(USE_QSSLDIFFIEHELLMANPARAMETERS)
+	if (! dhparams.isEmpty()) {
+		QSslDiffieHellmanParameters qdhp = QSslDiffieHellmanParameters(dhparams);
+		if (qdhp.isValid()) {
+			qsdhpDHParams = qdhp;
+		} else {
+			QSet<QSslDiffieHellmanParameters::Error> errors = qsdhpDHParams.errors();
```

Comment:
Well, this is my attempt at making it possible for the QSslDiffieHellmanParameters object to hold multiple errors at one time, and unfortunately it makes the API weird to work with.

Do you think it would be more workable to limit it to a single error at a time? Right now, the errors we can get are in fact exclusive, so I think it would make sense. I think in most cases in computer systems, you'll end up with the first encountered error anyway, when we're talking about error codes from a function or method.

So if I changed the API to:

    // Retrieve the error from an invalid (!isValid()) QSslDiffieHellmanParameters object.
    QSslDiffieHellmanParameters::Error QSslDiffieHellmanParameters:error();

   // Retrieve a human readable error string from QSslDiffieHellmanParameters.
   QString QSslDiffieHellmanParameters::errorString();

It'd be much easier to work with... WDYT?